gulpfile.js

```diff
@@ -64,6 +64,9 @@ gulp.task("scripts", function() {
     // Rename file
     .pipe(concat(paths.scripts.dest.files.unminified))
 
+    // Write un-minified version
+    .pipe(gulp.dest(paths.scripts.dest.dir))
```

Comment:
Moving this above ensures that the linting will occur on the dist file, not on the source files.